src/App.tsx

```diff
@@ -3,11 +3,28 @@ import './App.css';
 import {Flex} from 'react-flex';
 import 'react-flex/index.css';
 import {JamInput} from "./components/jamInput/JamInput";
+import {JamModal} from "./components/jamModal/jamModal";
 
 const logo = require('./img/logo.svg');
 const atom = require('./img/atom.png');
 
-class App extends React.Component {
+class App extends React.Component<IAppProps, IAppState> {
```

Comment:
I'm surprised you don't have to import the interfaces. Are they just globally available automatically?